ACKategoriesCore/RandomExtensions/String+Random.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+extension String: Randomizable {
+    public static func random() -> String {
+        String.random(minLength: 5)
+    }
+    
+    /// - Parameters:
+    ///     - minLength: Minimum length of a returned String
+    ///     - maxLength: Maxium length of a returned String
+    ///     - allowedLetters: Letters that will be used in the randomization
+    /// - Returns: Randomized String
+    public static func random(minLength: Int = 5,
```

Comment:
I am just wondering, if we have `Collection`s comforming to `Randomizable`, would it be possible to take advantage of `String`s being collections?